SharpPcap/LibPcap/SendQueue.cs

```diff
@@ -152,10 +152,8 @@ protected unsafe int ManagedTransmit(PcapDevice device, bool synchronized)
                     if (synchronized)
                     {
                         var timestamp = TimeSpan.FromTicks(header.Date.Ticks);
-                        while (sw.Elapsed < timestamp - firstTimestamp)
-                        {
-                            // Wait for packet time
-                        }
+                        var remainingTime = timestamp.Subtract(firstTimestamp);
+                        System.Threading.Thread.Sleep(remainingTime);
```

Comment:
@chmorgan This usually gives more accurate results on windows, could you give it a try on mac?
```suggestion
                        while (sw.Elapsed < timestamp - firstTimestamp)
                        {
                              // Wait for packet time
                              Thread.Sleep((timestamp - firstTimestamp)/ 2);
                        }
```

Reply:
@kayoub5 I couldn't find any numbers on sleep accuracy either way. How do you know that this timing gives better results?

Reply:
This is the best method we found that does not consume too much CPU (running some timer logic with millisecond precision).

For Micro/Nano second precision, the best thing I found was (this method consumes much more CPU than sleep)
https://stackoverflow.com/a/6259726/1438522

Reply:
Updated the changes to sleep with the incremental delay as suggested.